### DISCUSSION TITLE
List of things for the Daily Challenge

### DISCUSSION THREAD

**Discussion by Tenexxt** [ppy/osu#29078] [Ideas] [answered]
I really like it, its good and just by looking at how many people were satisfied at its relase, but there are some things that need to be added.

- **All SS scores get merged into one slot** - On the leaderboard its just a showcase of SS scores. Merge them into a expandable list that can be browsed, and all the scores in it are automatically tagged as 1st. After that list, show scores that come after that.
- **Freemod** - I know peppy said it will come mostlikely soon, **but its important that there should be a leaderboard with mods and without them.** This could be similar to how you can browse scores with specific mods on beatmaps. I don't know if this would be a supporter feature only too, but the idea is here nonetheless.
- **On score breakdown, when hovering onto a aprox. of the scores, display how many scores there are with such aprox.** - You know, when you hover on 400k for an example, display the ammount of scores with 400k
- Diffculty choose - self explainatory. Would be VERY usefull for many people. Don't get me wrong - the current system that makes the maps harder which each day isn't bad, but because of it many people won't be able to hold their streak - including me. While this is literally the definition of skill issue, not everyone want to play the game so much to learn how to play 6 star maps.
- Gamemode swap - My idea for this, is that you just add the converter to the DC. This way, the team behind choosing the map won't have to pick 3 more maps for a single day, but everyone will be happy... for the most part. Converted maps aren't as good as ones made specifically made for a mode, but this is a great option for some time.
- When in song select, the map that is for todays DC should be highlighted to tell you so - this is a small detail, but it would be cool and would make sense in my opinion.
- Streak leaderboards - to see who doesn't touch grass. The name explains this - basically a leader board of daily streaks, with weekly ones on the side with other DC stats
- osu!regular - a tilte for people who have a streak of 31+ days (aka a month). You could add a badge for it, and it would upgrade the longer your streak is, with the highest one being on 365 days (a year)

I have no other ideas for now, so give me your thoughts on this part. Also, i know this is more of a test of the DC, but im listing everything that could be added in the future

**Comment by bdach** [comment, marked as answer]
> Gamemode swap - My idea for this, is that you just add the converter to the DC.

I have very rarely - if ever - seen the other ruleset players approach converts with anything other than derision. From an implementation standpoint it's a compelling proposal *to me as a developer*, but I'm afraid for users this will just not fly.

> Streak leaderboards / custom titles

We've already come under criticism that a "daily" mechanic is "exploitative" or "gacha-like" even now that it's not yet linked to any substantial rewards other than a profile tooltip. I shudder to think what we'll hear if we add that.

**Reply by Tenexxt** [reply]
Yea, converted maps are worse than the ones made for that mode, but I'm saying that it would be a good solution for a limited time. Since the maps are picked by a team, and not automated - to balance things you'd have to atleast double that team so they don't have too much to do.
As for rewards, in my opinion you shouldn't listen too much to some of the opinions - possibly including mine. Especially when someone says that it's exploitave or gacha like - because people who say that dont know how a gacha works obviously. It will be hard to balance things out with rewards, but you'll have to come up with something eventually so people won't start complaining that there is nothing to get. In my opinion, hold a pool on the osu! Website about that so people can vote, but you'll also have to come up with something so it won't be botted
